src/test/java/resources/batch.feature

```diff
@@ -153,3 +153,16 @@ Feature: Batch Details API test
     When I call the batch summary API with expected status of 200
 	Then I am able to parse batch summary response
     And Status of transaction is "COMPLETED"
+
+  Scenario: Batch test for de-duplicating payments
+    Given I have the demo csv file "deduplication-test.csv"
+    And I have tenant as "gorilla"
+    And I have clientCorrelationId as "9051df83-e13c-4d6c-a850-220874db737a"
+    And I have private key
+    And I generate signature
+    When I call the batch transactions endpoint with expected status of 202
+    And I am able to parse batch transactions response
+    And I fetch batch ID from batch transaction API's response
+    And I call the batch details API with expected status of 200
```

Comment:
This step may not get the response immediately.  You have to add a retry mechanism with delay or call back to get this working.
The test will end up as flaky if this is not considered